### ISSUE TITLE
Allow customisation of the "Submit debug logs" modal

### ISSUE THREAD

**Issue by benparsons** [vector-im/element-web#22928]
#### What would you like to do?

Want to customise:

- The text in the description
- The default values in the fields

#### Why would you like to do it?

Not every installation goes back to Element (the org), some element-web installations are configured with their own homeserver.

#### How would you like to achieve it?

These values should be replaceable in config.json.

### Have you considered any alternatives?

Rather than being set in config.json, maybe this should be set by the homeserver? Where is the rageshake server itself selected?

**Comment by t3chguy** [comment]
The rageshake server is only configurable in config.json, having strings in config.json would require you to translate them which would very swiftly grow the size of the file for all supported languages given we don't support disabling languages

**Comment by benparsons** [comment]
Yes understood. So, is there any way to customise that screen? I suppose we can replace the screen wholesale? Would that be possible with https://github.com/matrix-org/matrix-react-sdk-module-api ?

**Comment by t3chguy** [comment]
@benparsons yes that would be in scope for the module API, but the module API is not generic in that way and a hook for this would need to be added. This sounds like a task for PS to pursue.
